acceptance/cli/cli.go

```diff
@@ -230,26 +230,26 @@ func setupKeys(ctx context.Context, vars map[string]string, environment []string
 }
 
 func setupSigs(ctx context.Context, vars map[string]string, environment []string) ([]string, map[string]string, error) {
-	type valFunc func(context.Context) (map[string]string, error)
+	type valFunc func(string, context.Context) (map[string]string, error)
```

Comment:
(nitpick) This would be more inline with golang style...

```suggestion
	type valFunc func(context.Context, string) (map[string]string, error)
```